test/e2e/examples.go

```diff
@@ -522,15 +523,22 @@ var _ = Describe("Examples e2e", func() {
 	})
 })
 
-func makeHttpRequestToService(c *client.Client, ns, service, path string) (string, error) {
-	result, err := c.Get().
-		Prefix("proxy").
-		Namespace(ns).
-		Resource("services").
-		Name(service).
-		Suffix(path).
-		Do().
-		Raw()
+func makeHttpRequestToService(c *client.Client, ns, service, path string, timeout time.Duration) (string, error) {
+	var result []byte
+	var err error
+	for t := time.Now(); time.Since(t) < timeout; time.Sleep(poll) {
```

Comment:
should probably use pkg/util/wait.Poll(). If the Get blocks, this function will wait indefinitely.